keywords.txt

```diff
@@ -177,6 +177,7 @@ MY_RS485_BAUD_RATE	LITERAL1
 MY_RS485_MAX_MESSAGE_LENGTH	LITERAL1
 MY_RS485_HWSERIAL	LITERAL1
 MY_RS485_DE_PIN	LITERAL1
+MY_RS485_SOH_COUNT LITERAL1
```

Comment:
The Arduino IDE is really stupid. It requires a **tab** character between "MY_RS485_SOH_COUNT" and "LITERAL1". At the moment you have a space character. Could you fix this?